tests/unit/completion/test_models.py

```diff
@@ -31,17 +31,18 @@
 from qutebrowser.config import sections, value
 
 
-def _get_completions(model):
-    """Collect all the completion entries of a model, organized by category.
-
-    The result is a list of form:
-    [
-        (CategoryName: [(name, desc, misc), ...]),
-        (CategoryName: [(name, desc, misc), ...]),
-        ...
-    ]
+def _check_completions(model, expected):
+    """Check that a model contains the expected items in any order.
+
+    Args:
+        expected: A list of form
```

Comment:
That sure is a weird list :wink: I can change it to "dict" while merging though if nothing else comes up.